pkg/pipelines/triggers/pipelinerun.go

```diff
@@ -18,7 +18,7 @@ func createDevCDPipelineRun(saName string) pipelinev1.PipelineRun {
 		Spec: pipelinev1.PipelineRunSpec{
 			ServiceAccountName: saName,
 			PipelineRef:        createPipelineRef("app-cd-pipeline"),
-			Resources:          createDevResource("$(params.gitsha)"),
+			Resources:          createDevResource("$(params.io.openshift.build.commit.id)"),
```

Comment:
Even param string like "$(params.io.openshift.build.commit.id)" has many re-occurances.   I wonder we should make them constants as well.